### DISCUSSION TITLE
"Beatmap failed to load" error keeps popping out

### DISCUSSION THREAD

**Discussion by WiIIiamWei** [ppy/osu#30593] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/787c2b4c-1fe8-4499-9018-0d705651a27a)

As the image shown above, this error pop-up keeps popping up every time I launch osu, even after I delete the beatmap or redownload it.

I assume this is an edge case for me and me only so I decided to open a discussion instead of an issue. I suppose a clean reinstall (with all user profile and data completely deleted) would fix, but redownloading gigabytes of beatmaps would be a pain in the ass. So I decided to ask here for a harmless solution.

The system is Windows 11, running version 2024.1009.1.

[Logs](https://github.com/user-attachments/files/17715920/compressed-logs.zip) here, in case needed.

**Comment by WiIIiamWei** [comment]
I suppose the related beatmap link is: https://osu.ppy.sh/beatmapsets/1841885#osu/3783212

**Comment by SupDos** [comment]
I believe that's the intro/welcome music that has somehow been deleted from your database. If you choose a different intro sequence or turn it off does that stop the error message from popping up when you launch the game?

note that this is not a solution, just to determine if that is actually the issue or not

**Reply by WiIIiamWei** [reply]
The error indeed disappear when I change my Welcome animation to Circles instead of Triangles. Just why the Intro music (triangles) would be "missing"? I do hear it when I use the Triangles Welcome animation.

**Reply by sineplusx** [reply, marked as answer]
You can probably get rid of the error by just importing [the map](https://github.com/ppy/osu-resources/blob/master/osu.Game.Resources/Tracks/triangles.osz) again. The file missing is the (empty) beatmap file for the triangles intro.

**Comment by Loreos7** [comment]
Related: https://github.com/ppy/osu/discussions/22185
